### PR TITLE
resolve conflicts of component library

### PR DESCRIPTION
**Pull request by arpit01923**
Files changed
- Fix card with the badge, the card with dismiss 
-  Card with Text Overlay 
-  Add better example of responsive image 
-  Add variant: input with validation 
-  Add heading classes and use them in heading tags to make heading text utility 
-  Add a better example of stacked List 
-  Fix variant: Add proper example of Toast, current version is more like modal

### CODE REVIEW

File: css/components/list.css [hunk 1]
```diff
@@ -21,3 +21,14 @@
     border-bottom: 1px solid var(--default);
     list-style: none;
 }
+
+.list-item-image{
+    object-fit: cover;
+    width: 50px;
+    height: 50px;
```
Comment (pradipgharge) [inline css/components/list.css]: use rems here

**Review by pradipgharge — APPROVED** [review]

File: css/components/list.css (hunk 1 above)
Comment (chanchal16) [reply css/components/list.css]: Would recommend using rems to make image responsive.

File: documentation/docs.html [hunk 2]
```diff
@@ -1176,16 +1206,28 @@ <h2>Example of List</h2>
           <section class="flex">
             <div class="border">
               <ul class="stacked-list">
-                    <li class="list-item">List Item 1</li>
-                    <li class="list-item">List Item 2</li>
-                    <li class="list-item">List Item 3</li>
-                    <li class="list-item">List Item 4</li>
-              </ul>
+        <li class="list-item">
```
Comment (chanchal16) [inline documentation/docs.html]: Use prettier to format the code.

**Review by chanchal16 — APPROVED** [review]
- your code looks clean
- liked that you gave meaningful and readable class names.
Good to merge!

**Review by bajajcodes — COMMENTED** [review]
- Code style is consistent.
- File naming is not consistent check in css folder.
- Use rems for values bigger than then what you have decided, such as 16px.
- Remove commented code.

File: css/components/list.css (hunk 1 above)
Comment (arpit01923) [reply css/components/list.css]: changed

**Pull request merged**